### DISCUSSION TITLE
Get ratio of image with NuxtImage for explicit width and height (cls)

### DISCUSSION THREAD

**Discussion by fabianwohlfart** [nuxt/image#1009] [Q&A]
When working with NuxtImage I work with images where I do not know the height, because they come from a editor / cms in an article.
How do I know the aspect ratio for an image so i can any evade any cumulative layout shift?
[Nuxt-Img: Width/Height](https://image.nuxt.com/usage/nuxt-img#width-height)
Lighthouse: "Image Elements do not have explicit width and height"

```
<NuxtImg
   :src="src"
   width="1000"
   height="???"
   loading="lazy"
   />

---

const img = useImage()

const sources = computed(() => {
  return img.getSizes(props.src)
})

console.log('height?', sources.ratio)

```

**Comment by DavidDeSloovere** [comment]
I don't have an answer to your question, but maybe an alternative:
You could force an aspect ratio and crop images.
If you document this for the content editors, they can keep this in mind.
We use DatoCMS and it has an option to validate aspect ratio of selected images. It also has [thumbhash](https://www.datocms.com/blog/introducing-thumbhash-the-compact-placeholder-with-alpha-support) which is awesome.

**Comment by luksak** [comment]
I've been trying to get this working for a while without success so far.  Here is a related discussion: https://github.com/johannschopplich/unlazy/discussions/11

**Comment by luksak** [comment]
Ok, I've got a working solution for `@nuxt/content` in https://github.com/attributech/nuxt-attribute-content. See my comment here: https://github.com/johannschopplich/unlazy/discussions/11#discussioncomment-14839025
